en/ui-system/components/editor/safearea.md

```diff
@@ -12,6 +12,8 @@ It is strongly discouraged from providing controls outside of this area. But the
 
 The developer only needs to add the component on the node, without any settings. This component internally uses the API cc.sys.getSafeAreaRect(); to obtain the safe area of the current iOS or Android device and implements the adaptation by using the Widget component and set anchor. (If there is no Widget component, it will be added automatically)
 
+Api can refer to [SafeArea API](https://docs.cocos.com/creator3d/api/en/classes/ui.safearea.html).
```

Comment:
Api can refer to [SafeArea API](https://docs.cocos.com/creator3d/api/en/classes/ui.safearea.html).
-->
Please refer to the [SafeArea API](https://docs.cocos.com/creator3d/api/en/classes/ui.safearea.html).